e2e/cypress/integration/enterprise/guest_accounts/member_invitation_ui_spec.js

```diff
@@ -203,7 +203,7 @@ describe('Guest Account - Member Invitation Flow', () => {
         invitePeople(user1.email, 1, user1.username);
 
         // * Verify the content and message in next screen
-        verifyInvitationSuccess(user1.username, 'This member has been added to the team.');
+        verifyInvitationSuccess(user1.username, 'An invitation email has been sent.');
```

Comment:
The change here is failing for me, whereas the former assertion with "This member has been added to the team" works.

Reply:
Seeing that as well
![Screen Shot 2020-01-31 at 10 37 00 AM](https://user-images.githubusercontent.com/5334504/73508045-c480d180-4415-11ea-9833-6970a921213d.png)

Reply:
Updated